feature_engine/tests/test_missing_data_imputer.py

```diff
@@ -1,160 +1,111 @@
 # Authors: Soledad Galli <solegalli1@gmail.com>
 # License: BSD 3 clause
 
-import pandas as pd
+import os
+
 import numpy as np
-import pytest
+import pandas as pd
 
-import os
 filename = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'titanic_test.csv')
 
-from missing_data_imputation import MeanMedianImputer
-from missing_data_imputation import RandomSampleImputer
-from missing_data_imputation import EndTailImputer
-from missing_data_imputation import na_capturer
-from missing_data_imputation import CategoricalImputer
-from missing_data_imputation import ArbitraryImputer
+from feature_engine.missing_data_imputers import MeanMedianImputer
+from feature_engine.missing_data_imputers import RandomSampleImputer
+from feature_engine.missing_data_imputers import EndTailImputer
+from feature_engine.missing_data_imputers import CategoricalVariableImputer
+
 
 df_test = pd.read_csv(filename)
 
 def test_MeanMedianImputer():
-    imputer = MeanMedianImputer(imputation_method='median')
-    imputer.fit(df_test, variables = ['Age'])
+    imputer = MeanMedianImputer(imputation_method='median', variables=['Age'])
+    imputer.fit(df_test)
     X = imputer.transform(df_test)
     df_transf = df_test.copy()
     df_transf['Age'] = df_test['Age'].fillna(31.5)
     pd.testing.assert_frame_equal(X, df_transf)
-    assert imputer.variables_ == ['Age']
```

Comment:
why did you remove this one? can we not keep:
assert imputer.variables == ['Age']